src/shared/constants/index.js

```diff
@@ -16,4 +16,8 @@ export const Constants = {
   DEFAULT_NODEPOOL_NAME: 'Unnamed node pool',
   AWS_V5_VERSION: '10.0.0',
   AZURE_MULTI_AZ_VERSION: '11.1.0',
+
+  // UI labels
+  DESIRED_NODES_EXPLANATION:
+    'Autoscaler&apos;s idea how many nodes would be required for the workloads',
```

Comment:
Backticks would do the trick:

```suggestion
`Autoscaler's idea how many nodes would be required for the workloads`
```

Reply:
Maybe, but the character you are suggesting there is not the apostrophe I want. It's a single quotation mark, which doesn't make for typographically appealing apostrophe.

Reply:
I could just use the unicode character, but think this here is more robust, no?

Reply:
Damn, now I see that it actually fails to render properly.... didn't even look at my own screenshot.

Reply:
The grammar here was a bit weird for me, but I was too late to comment.

I think it should've been "Autoscaler's idea _of_ how many many..."